src/components/propertydetails/Review.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import "./propertydetails.css";
+
+function Review() {
+  return (
+    <div className="review-container">
+      <div className="email-agent-container">
+        <span className="email-agent-header">
+          <h4>More about Property</h4>
+        </span>
+        <div className="contact-agent-wrapper">
+          <div className="input-container">
+            <i className="fa fa-user form-icon"></i>
+            <input className="field" type="text" placeholder="Username" />
+          </div>
```

Comment:
I suggest the input container in contact-agent-wrapper can be rendered as a component and map through so as to stick with DRY principles